datadog_checks_downloader/README.md

```diff
@@ -50,6 +50,21 @@ To run the tests, [install tox][8] and just run:
 tox
 ```
 
+You can select between online and offline tests when running testsuite using
+pytest:
+
+```shell
+pytest -vvvv -m online     # Run tests that use data from publicly a accessible repository.
+pytest -vvvv -m offline    # Run tests that use data stored in the Git repository.
```

Comment:
`pytest` does not seem to be in the dev dependencies, should we add it?

Reply:
It should be part of dev dependencies. See the installation instructions, let me know if it will not work.